src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -208,17 +209,86 @@
 
   describe 'GET #new' do
     before do
-      login(user_moi)
+      login user_moi
       get :new, name: 'ProjectName'
     end
 
     it { expect(assigns(:project)).to be_a(Project) }
     it { expect(assigns(:project).name).to eq('ProjectName') }
   end
 
+  describe 'GET #show' do
+    before do
+      Project.any_instance.stubs(:number_of_build_problems).returns(0) # To not ask backend for build status
```

Comment:
Hmmmm. This sounds like a good default for the whole test suite? @bgeuken @ChrisBr what do you think?